includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -624,27 +624,31 @@ public function get_header_logo_height() {
 	 * Show logo html
 	 */
 	public function header_logo() {
-		if ($this->get_header_logo_id()) {
+		if ( $this->get_header_logo_id() ) {
 			$attachment_id = $this->get_header_logo_id();
-			$company = $this->get_shop_name();
-			if( $attachment_id ) {
-				$attachment = wp_get_attachment_image_src( $attachment_id, 'full', false );
+			$company       = $this->get_shop_name();
+
+			if ( $attachment_id ) {
+				$attachment      = wp_get_attachment_image_src( $attachment_id, 'full', false );
 				$attachment_path = get_attached_file( $attachment_id );
+
 				if ( empty( $attachment ) || empty( $attachment_path ) ) {
 					return;
 				}
 				
-				$attachment_src = $attachment[0];
-				$attachment_width = $attachment[1];
+				$attachment_src    = $attachment[0];
+				$attachment_width  = $attachment[1];
 				$attachment_height = $attachment[2];
 
 				if ( apply_filters( 'wpo_wcpdf_use_path', true ) && file_exists( $attachment_path ) ) {
 					$src = $attachment_path;
-				} else {
+				} elseif ( wp_remote_head( $attachment_src, [ 'sslverify' => false ] )['response']['code'] === 200 ) {
```

Comment:
I think you probably should check the response first before asking for the keys, like this:

```php
elseif( ( is_array( $head = wp_remote_head( $attachment_src, [ 'sslverify' => false ] ) ) ) && $head['response']['code'] == 200 ) )
```

Reply:
Done here: 0601f05